pyEpiabm/pyEpiabm/core/cell.py

```diff
@@ -90,7 +90,8 @@ def enqueue_person(self, person: Person):
             Person to enqueue
 
         """
-        self.person_queue.put(person)
+        if person.infection_status != InfectionStatus.Recovered:
```

Comment:
Agreed, just need to note this restriction in the docstring since its not always obvious from the name of the method